server/controllers/middlewares.js

```diff
@@ -257,4 +258,17 @@ export const sorting = (options) => {
 
     next();
   };
+};
+
+/**
+ * Populate req with info on whether this user can edit colletive
+ */
+export const populateEditPermissions = (req, res, next) => {
+  if (req.remoteUser && req.group) {
+    return req.group.hasUserWithRole(req.remoteUser.id, [roles.HOST, roles.MEMBER])
+    .then(canEditGroup => req.remoteUser.canEditGroup = canEditGroup)
```

Comment:
I also wonder how this will work with graphql because `req.group` is not going to be populated.